cirq/ion/ion_decomposition_test.py

```diff
@@ -0,0 +1,136 @@
+import random
+
+import numpy as np
+import pytest
+
+import cirq
+
+
+def _operations_to_matrix(operations, qubits):
+    return cirq.Circuit.from_ops(operations).to_unitary_matrix(
+        qubit_order=cirq.QubitOrder.explicit(qubits),
+        qubits_that_should_be_present=qubits)
+
+
+def _random_single_MS_effect():
+    t = random.random()
+    s = np.sin(t)
+    c = np.cos(t)
+    return cirq.dot(
+        cirq.kron(cirq.testing.random_unitary(2),
+                  cirq.testing.random_unitary(2)),
+        np.array([[c, 0, 0, -1j*s],
+                  [0, c, -1j*s, 0],
+                  [0, -1j*s, c, 0],
+                  [-1j*s, 0, 0, c]]),
+        cirq.kron(cirq.testing.random_unitary(2),
+                  cirq.testing.random_unitary(2)))
+
+
+def _random_double_MS_effect():
```

Comment:
I think it's okay if you import the equivalent of these methods from the other decomposition test file. The assertions about the CZ count and etc should still hold.

Reply:
Here Im testing for MS gates and effects so I cannot use the methods written for CZs.